backend/src/services/certificate-authority/certificate-authority-queue.ts

```diff
@@ -71,16 +114,76 @@ export const certificateAuthorityQueueFactory = ({
     );
   };
 
+  const orderCertificateForSubscriber = async ({ subscriberId, caType }: TOrderCertificateForSubscriberDTO) => {
```

Comment:
Just to confirm, we're not doing ACME certificate auto-renewal in this iteration right? (i.e. fetching new certificate prior to old one expiring such that getting latest bundle for the corresponding subscriber will always result in a valid, not-expired certificate).

Reply:
yup, no auto-renewals yet

Reply:
just do this